web/controllers/notifier_controller.ex

```diff
@@ -24,6 +24,7 @@ defmodule Notifilter.NotifierController do
     {:ok, notifier_as_json} = notifier
     |> Map.from_struct
     |> Map.put(:eventName, notifier.event_name)
+    |> Map.put(:previewOffset, 0)
```

Comment:
You also need to add this into `new`.

Maybe it's a better idea to have this key set as a default in React, not loaded in from the HTML page. It's not really related to the notifier itself, only when previewing. WDYT?